TestUtilities/README.md

```diff
@@ -0,0 +1,3 @@
+# TestUtilities
+
+Package that provides foundational tools, extensions and, mocks needed for testing other targets.
```

Comment:
```suggestion
Package that provides foundational tools, extensions, and mocks needed for testing other targets.
```

Reply:
TY! Changed ✅ - we will merge it without awaiting for re-approval